src/index.ts

```diff
@@ -30,6 +33,8 @@ import { parseSdkInfoResponse } from "./0xsquid/v1/sdk-info";
 import { parseRouteResponse } from "./0xsquid/v1/route";
 import { parseStatusResponse } from "./0xsquid/v1/status";
 
+const appsSdk = new SafeAppsSDK({ debug: true });
```

Comment:
Aren't we supposed to provide 'allowedDomains: [/gnosis-safe.io$/, /app.safe.global$/],' as well ?

Reply:
Plus can't we pass this to the function from whatever we get from wagmi ? @odcey

Reply:
> Aren't we supposed to provide 'allowedDomains: [/gnosis-safe.io$/, /app.safe.global$/],' as well ?

Agree, it's better to add this as an option in the constructor